mlrun/api/crud/model_endpoints.py

```diff
@@ -967,25 +660,414 @@ def deploy_model_monitoring_batch_processing(
         )
 
     @staticmethod
-    def get_endpoint_features(
-        feature_names: typing.List[str],
-        feature_stats: typing.Optional[dict],
-        current_stats: typing.Optional[dict],
-    ) -> typing.List[mlrun.api.schemas.Features]:
-        safe_feature_stats = feature_stats or {}
-        safe_current_stats = current_stats or {}
+    def _clean_feature_name(feature_name):
+        return feature_name.replace(" ", "_").replace("(", "").replace(")", "")
 
-        features = []
-        for name in feature_names:
-            if feature_stats is not None and name not in feature_stats:
-                logger.warn(f"Feature '{name}' missing from 'feature_stats'")
-            if current_stats is not None and name not in current_stats:
-                logger.warn(f"Feature '{name}' missing from 'current_stats'")
-            f = mlrun.api.schemas.Features.new(
-                name, safe_feature_stats.get(name), safe_current_stats.get(name)
+    @staticmethod
+    def get_access_key(auth_info: mlrun.api.schemas.AuthInfo):
+        access_key = auth_info.data_session
+        if not access_key:
+            raise mlrun.errors.MLRunBadRequestError("Data session is missing")
+        return access_key
+
+    @staticmethod
+    def _get_batching_interval_param(intervals_list):
+        """Converting each value in the intervals list into a float number. None
+        Values will be converted into 0.0.
+
+        example::
+        Applying the function on a scheduling policy that is based on every hour exactly
+        _get_batching_interval_param(intervals_list=[0, '*/1', None])
+        The result will be: (0.0, 1.0, 0.0)
```

Comment:
Although this function was merged in https://github.com/mlrun/mlrun/pull/2220 , I'll remove the example and fix the docs here, so it will be more readable.